src/script/ceph-release-notes

```diff
@@ -105,9 +105,9 @@ def split_component(title, gh, number):
         issue_labels = {it['name'] for it in issue['labels']}
         if 'documentation' in issue_labels:
             return 'doc: ' + title
-        item = labels.intersection(issue_labels)
+        item = (labels.union(set(prefixes))).intersection(issue_labels)
```

Comment:
ack, this doesn't make sense anymore, my idea was to find the cases where we have a proper title and no labels, but these are already caught before.